### DISCUSSION TITLE
Error: The correlation name 'MyTable' has the same exposed name as table 'MyTable'

### DISCUSSION THREAD

**Discussion by lukuser** [Budibase/budibase#8900] [Help]
I'm using an Azure SQL data source with several tables and multiple relationships. When I try to autogenerate a screen out of one of the tables, I see an error repeated several times:

_"The correlation name 'MyTable' has the same exposed name as table 'MyTable'._

The item (:id) screen doesn't have any components on it. When the form block is ejected on it, there's no components below the first repeater. The New and List screens seem to be ok.

**Comment by melohagan** [comment]
Hey @lukuser 

When you say Azure SQL datasource, is that SQL Server or something else? 

Would you be able to provide the SQL scripts to generate the tables and some data. Also having some screenshots of how you have defined the relationships could be useful too.
